src/pcp2json/pcp2json.py

```diff
@@ -151,7 +157,7 @@ def options(self):
         opts = pmapi.pmOptions()
         opts.pmSetOptionCallback(self.option)
         opts.pmSetOverrideCallback(self.option_override)
-        opts.pmSetShortOptions("a:h:LK:c:Ce:D:V?HGA:S:T:O:s:t:rRIi:jJ:4:58:9:nN:vmP:0:q:b:y:Q:B:Y:F:f:Z:zxXE")
+        opts.pmSetShortOptions("a:h:LK:c:Ce:D:HGA:S:T:O:s:t:rRIi:jJ:4:58:9:nN:vmP:0:q:b:y:Q:B:Y:F:f:Z:zxXEp:U:u:V?")
```

Comment:
IIRC there was some sort of logic to keep "common parameters" at the beginning followed by "mostly common" and then lastly tool specific options but I don't think anyone remembers that logic anymore.

Reply:
Ah, I see the pattern - fixed now.